tests/query/allocation_policies/test_bytes_scanned_rejecting_policy.py

```diff
@@ -291,3 +244,63 @@ def test_penalize_timeout(policy: BytesScannedRejectingPolicy) -> None:
 
     allowance = policy.get_quota_allowance(tenant_ids, QUERY_ID)
     assert not allowance.can_run
+
+
+@pytest.mark.parametrize(
+    ("tenant_ids", "bytes_to_scan", "limit"),
+    [
+        pytest.param(
+            {
+                "project_id": 4505240668733440,
+                "referrer": "api.trace-explorer.stats",
+            },
+            100000001,
+            200000000,
+        ),
+    ],
+)
```

Comment:
You probably don't need to parametrize this, there's only one example and it's testing a specific behavior